source/dev-setup.rst

```diff
@@ -59,6 +59,12 @@ extensions::
     brew install postgres
     brew install postgis
 
+Note that "brew install postgis" will install the latest version of postgis along with the latest
+version of potstgres, so if you have a previous installation of postgres you may end up with two
+versions installed causing conflicts and connection issues. It is recommended, if you have a
+previous installation, that you uninstall this version version. Migration of existing data is
+left as an exercise for the reader.
```

Comment:
I'm not sure 1) that its needed for developer machines or that 2) we want to include instructions we haven't actually vetted and walked through. I'd feel more comfortable documenting more sure-fire, if destructive, options. Do you feel comfortable including unvetted instructions?